consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/request/claim/ClaimRequest.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.moj.sdt.cmc.consumers.request.claim;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+@Data
+@ToString
+@NoArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonPropertyOrder({
+    "claimantReference",
+    "claimant",
+    "defendant1",
+    "defendant2",
+    "correspondenceDetailType",
```

Comment:
Does this need to be renamed to "claimantCorrespondence" to match the name of the field?

Reply:
done